layouts/robots.txt

```diff
@@ -0,0 +1,5 @@
+User-agent: *
+
+{{range .Data.Pages}}
+{{end}}
```

Comment:
What is this loop doing here? :confused:

Reply:
Looping trough pages for no reason. 😄 😃